cli/sync_service/README.md

```diff
@@ -0,0 +1,362 @@
+# Model Management System
+
+## Introduction
+The model management system (MMS) is one of the most important parts in Edge Fabric. It is leveraged to ease the burden of AI model management of cognitive services running on an edge node. Edge Fabric provides a set of hzn CLIs to use MMS to manipulate the models object and its metadata.
+
+## MMS CLI
+ 
+### Check MMS status
+
+Before pushing the object, we need to check MMS status using `hzn mms status` command, to make sure MMS is running properly. Check `heathStatus` under `general` and `dbStatus` under `dbHealth`. The values of these two fields should be “green”, which indicate that CSS (cloud sync service part of MMS) and database are both running.
+
+    root@lily-test:~/go/src/github.com/open-horizon/anax# hzn mms status
+    {
+      "general": {
+        "nodeType": "CSS",
+        "healthStatus": "green",
+        "upTime": 21896
+      },
+      "dbHealth": {
+        "dbStatus": "green",
+        "disconnectedFromDB": false,
+        "dbReadFailures": 0,
+        "dbWriteFailures": 0
+      }
+    }
+    
+### Create and publish MMS object
+
+#### Create MMS Object
+
+In MMS, your model file is not published by its own. MMS requires a metadata file along with your model file when publishing and distributing your model file. Metadata file configures a set of attributes of your model. MMS will store, distribute, and retrieve the model objects based on those attributes defined in metadata. 
+
+Metadata file is a json file. You can use `hzn mms object new` to get a template of metadata file. 
+
```

Comment:
The full output from object new will get stale over time, so we should omit it. But, you will need a section where you can describe what each field does. Maybe that's in a reference or appendix section at the end.

Reply:
Done